### ISSUE TITLE
Create a Flatpak when a release is published

### ISSUE THREAD

**Issue by Nick-Hall** [gramps-project/flatpak#6]
It appears that we can run a workflow when a release is published.  See the [documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onevent_nametypes) on events for details.

**Comment by OzarkShepherd** [comment]
The reason I made the workflow run on push during the writing of the flatpak manifest was because I wanted to download the resulting archive to test the changes to make sure they worked on one of my computers, but I could see how running a workflow on each push could become undesirable.  I would expect the flatpak will need to be tested and perhaps modified when Gramps 5.2 comes out, before pushing the upgrade to flathub.

**Comment by OzarkShepherd** [comment]
Would this model of publishing releases be acceptable?
https://github.com/gramps-project/flatpak/releases

I did about 70 commits between 5.1.6-1 and 5.2.0-rc1. Most of them were troubleshooting dependency updates, but some were normal changes for newer runtimes and such that have to be worked through with every release. I doubt that any time a workflow is automatically triggered for a new Gramps release, that a functional flatpak will get created because other changes usually are required too.

After several years, I expect that Flathub will remove Gnome Runtime 45 and other required runtimes from its public repository, so the 5.2.0-rc1 release will likely stop being installable at that time.

**Comment by Nick-Hall** [comment]
Yes, that's perfectly acceptable.  Thanks for your efforts in maintaining this.
